app/ads/components/mobile-my-ads-list.tsx

```diff
@@ -135,16 +131,18 @@ export default function MobileMyAdsList({ ads, onAdDeleted }: MobileMyAdsListPro
   }
 
   const handleDelete = (adId: string) => {
-    setDeleteConfirmModal({
-      show: true,
-      adId: adId,
+    showAlert({
+      title: "Delete ad?",
+      description: "You will not be able to restore it.",
+      confirmText: "Delete",
+      onConfirm: () => confirmDelete(adId),
     })
   }
 
-  const confirmDelete = async () => {
+  const confirmDelete = async (adId: string) => {
     try {
       setIsDeleting(true)
-      const result = await deleteAd(deleteConfirmModal.adId)
+      const result = await deleteAd(adId)
 
       if (result.errors && result.errors.length > 0) {
         const errorMessage = result.errors[0].message || "Failed to delete ad. Please try again."
```

Comment:
we should sync later regarding error messages, just to make sure all our error messages are aligned